components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.tsx

```diff
@@ -116,6 +118,15 @@ export default class PermissionsTree extends React.PureComponent<Props, State> {
                     Permissions.DELETE_PRIVATE_CHANNEL,
                 ],
             },
+            {
+                id: 'playbook_public',
+                permissions: [
+                    Permissions.PLAYBOOK_PUBLIC_CREATE,
+                    Permissions.PLAYBOOK_PUBLIC_MANAGE_PROPERTIES,
+                    Permissions.PLAYBOOK_PUBLIC_MANAGE_MEMBERS,
+                ],
+                isVisible: isNonEnterpriseLicense,
```

Comment:
Will these not be visible if you do have an Enterprise license?

Reply:
Admittedly, the logic here is a bit subtle -- we actually want `playbook_public` to be visible for all users, but only enterprise customers have access to the `Permissions.PLAYBOOK_PUBLIC_MAKE_PRIVATE` permission. Instead of annotating each permission with an `isVisible`, the logic here duplicates the two blocks and configures each separately.

Reply:
Right but wouldn't `isNonEnterpriseLicense` be `false` if they have the `Enterprise` sku, hiding this section for them?

Reply:
Indeed, this version of `playbook_public` gets hidden for enterprise customers, but they see the one defined on line 131 instead. (And vice-versa for non-enterprise customers).

I'm thinking I need to do something to document this more clearly :/

Reply:
Gotcha, didn't notice it was included in the other sections below. Makes sense now. Thanks!